cpp/src/Ice/ObjectAdapterI.cpp

```diff
@@ -934,7 +934,7 @@ Ice::ObjectAdapterI::initialize(optional<RouterPrx> router)
         //
         if (router == nullopt && noProps)
         {
-            throw InitializationException(__FILE__, __LINE__, "object adapter `" + _name + "' requires configuration");
+            throw InitializationException(__FILE__, __LINE__, "Object adapter `" + _name + "' requires configuration.");
```

Comment:
Please revert all these updates.

Reply:
Alright, now that I have a bearing for what the intent is, I'll revert/fix my changes as necessary.
Unfortunately, we're very inconsistent about which message style we use across our languages, so extrapolated the wrong idea.

One day we should consider a cleanup pass to fix all these messages...

Reply:
I fixed all my changes to the messages, not by reverting them, but by changing them to be correct in each language.
Since most of them didn't follow the guidelines you laid out...